tests/quality/configure.py

```diff
@@ -601,8 +615,8 @@ def build_db(cfg: Config):
         subprocess.run(["vunnel", "-v", "run", provider], check=True)
 
     logging.info("building DB")
-    subprocess.run([GRYPE_DB, "build", "-v"], check=True)
-    subprocess.run([GRYPE_DB, "package", "-v"], check=True)
+    subprocess.run([GRYPE_DB, "build", "-v", "-c", ".grype-db.yaml"], check=True)
```

Comment:
@wagoodman this is part of the fix that I wanted your feedback on. My understanding of how `grype-db` searches for config files says I shouldn't have to pass that, but it fails unless I do (It seems to find the `.grype-db.yaml` that gets written at the repo root rather than the one that gets written in `tests/quality`.